packages/List/src/list-item.scss

```
@@ -0,0 +1,274 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-list-item-padding: #{tokens.$space-2} #{tokens.$space-3};
+  --ids-list-item-min-height: #{tokens.$space-8};
+
+  /* Compact */
+  --ids-list-item-padding-compact: #{tokens.$space-1} #{tokens.$space-3};
```

The padding top and bottom is 8px;

```suggestion
  --ids-list-item-padding-compact: #{tokens.$space-2} #{tokens.$space-3};
```